lib/index.js

```diff
@@ -204,11 +204,11 @@ Nightwatch.cli = function(callback) {
   } else if (argv['list-files']) {
     argv._source = argv['_'].slice(0);
     const runner = Nightwatch.CliRunner(argv);
-    
-    runner.setupAsync()
+
+    return runner.setupAsync()
```

Comment:
if you add `return` here, you should delete the `else`

Reply:
IMO, it's not required as this condition will only return in case of `list-files`; for every other case, `else` may execute

Reply:
there is no reason to add the return here, but if you do you need to refactor the rest of the code because else after return is not useful.

Reply:
@beatfactor This might me a stupid question, but in the case that any of the following flags are not provided (`help`, `info`, `version`, `list-files`), then else should be called right? Even if there is a return statement.

Reply:
yes, which is why I mentioned the refactoring. We need to write this something like:

```js
Nightwatch.cli = function(callback) {
  const ArgvSetup = require('./runner/cli/argv-setup.js');
  const {argv} = ArgvSetup;

  if (argv['list-files']) {
    argv._source = argv['_'].slice(0);
    const runner = Nightwatch.CliRunner(argv);
    
    return runner.setupAsync();
  }

  if (argv.help) {
  // etc...
}   
```

Reply:
Got it

Reply:
Thanks, updated the PR